impeller/entity/contents/text_contents.cc

```diff
@@ -58,6 +58,14 @@ std::optional<Rect> TextContents::GetCoverage(const Entity& entity) const {
   return bounds->TransformBounds(entity.GetTransformation());
 }
 
+static Vector4 PositionForGlyphPosition(const Matrix& translation,
+                                        Point unit_position,
+                                        Size destination_size) {
+  return translation * Vector4(unit_position.x * destination_size.width,
+                               unit_position.y * destination_size.height, 0.0,
+                               1.0);
```

Comment:
Doesn't `translation * (unit_position * destination_size)` work fine for this, or are we missing an overload somewhere? Size should get coerced to `Point`.

Reply:
Done